subsystems/win32/csrsrv/server.c

```diff
@@ -676,8 +679,13 @@ CsrUnhandledExceptionFilter(IN PEXCEPTION_POINTERS ExceptionInfo)
                                       ErrorParameters,
                                       OptionShutdownSystem,
                                       &Response);
+            if (!NT_SUCCESS(Status))
```

Comment:
What for? The CSRSS is killed just afterwards, so there is no real point in telling why NtRaiseHardError could have failed. (It's not like if we wanted to then do actions depending on the returned status.)

Reply:
Reverted, then.